### PR TITLE
Remove unused level_source_id from appOptions

### PR DESCRIPTION
**Pull request by balderdash**
The level_source_id that actually gets used is later fetched via ajax from /api/user_progress. The one in the appOptions object is ignored.

### CODE REVIEW

File: dashboard/test/controllers/level_sources_controller_test.rb [hunk 1]
```diff
@@ -167,22 +167,6 @@ def expect_s3_upload
     assert_equal "max-age=36000, public", response.headers["Cache-Control"]
   end
 
-  test 'include level source ID for send to phone dialog' do
-    # Prevents regressions in #79201066
```
Comment (balderdash) [inline dashboard/test/controllers/level_sources_controller_test.rb:171]: AFAICT, appOptions.level_source_id isn't used by the share dialog anymore, and the dialog works for me locally with artist and applab. Is there some other project type that could still be using it?

**Comment by wjordan** [comment]
Yeah, it's possible that this setting and regression-test has been made obsolete by a more recent code change (if so, I don't think I was involved in the change). So before removing this code, I'd like us to track down the actual commit/PR where the change occurred (and run this by whoever made that change), just to make sure the new implementation did completely replace what this code was written for in the first place.

**Comment by balderdash** [comment]
@aoby removed the last use of it in https://github.com/code-dot-org/code-dot-org/pull/5850, but it was dead code since https://github.com/code-dot-org/code-dot-org/pull/5777.

Andrew, am I good to remove `level_source_id` from appOptions, at least as far as that share view is concerned?

**Comment by aoby** [comment]
Yes if that's the only place it was used. The share dialog (at least as of those PRs you linked - I haven't been focusing on that area since) did not use it, and if you can view it successfully then 👍

**Comment by wjordan** [comment]
ok, sounds good to me! :+1:

**Pull request merged**